src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+function promise1() {
+  return new Promise((resolve) => {
+    const logo = document.getElementById('logo')
+    const message1 = document.getElementById('message1')
+
+    logo.addEventListener('click', () => {
+      resolve(message1.innertext = 'Promise was resolved!')
+    })
+  })
+}
```

Comment:
promise1 - is not a function. it should be a variable which equals returned promise
and then you shoud use promise1.then() syntax to catch resolve method and promise1.catch() to catch an error